addr2line_test.cc

```diff
@@ -0,0 +1,18 @@
+#include "addr2line.h"
+#include "third_party/abseil/absl/strings/str_cat.h"
+#include "util/testing/status_matchers.h"
+
+namespace {
+
+using ::devtools_crosstool_autofdo::Addr2line;
+
+TEST(Addr2lineTest, ComdatFuncHasNoDwp) {
```

Comment:
Should we change the name of the test? I don't think this has anything to do with Comdats :)

Reply:
Good catch, fixed.